pyproject.toml

```diff
@@ -52,7 +52,7 @@ dependencies = [
     "pandas >=1.5",
     "scipy>=1.8",
     "seaborn>=0.13",
-    "h5py>=3.6",
+    "h5py>=3.7",
```

Comment:
h5py 3.6.0 doesn’t build on macOS M1, 3.7.0 has wheels.

Idk if that’s a good reason to bump the min deps …